.github/dependabot.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+updates:
+- package-ecosystem: docker
+  directory: /cluster/gce/gci/mounter
+  schedule:
+    interval: weekly
+- package-ecosystem: docker
+  directory: /cluster/addons/addon-manager
+  schedule:
+    interval: weekly
+- package-ecosystem: gomod
+  directory: /providers
+  schedule:
+    interval: weekly
+  groups:
+    gomod-dependencies:
+      patterns:
+      - '*'
+- package-ecosystem: gomod
+  directory: /
+  schedule:
+    interval: weekly
+  groups:
+    gomod-dependencies:
+      patterns:
+      - '*'
+- package-ecosystem: gomod
+  directory: /crd
+  schedule:
+    interval: weekly
+  groups:
+    gomod-dependencies:
+      patterns:
+      - '*'
+- package-ecosystem: gomod
+  directory: /crd/hack
+  schedule:
+    interval: weekly
+  groups:
+    gomod-dependencies:
+      patterns:
+      - '*'
```

Comment:
you can add the labels like ok-to-test to start ci like we do https://github.com/kubernetes/release/blob/master/.github/dependabot.yml#L7-L10

Reply:
hello, sorry for not timely updating the pr. added the requested tags for all update subsections. please give a re-review:)